test/models/spicepod_hf.yml

```diff
@@ -0,0 +1,81 @@
+version: v1beta1
+kind: Spicepod
+name: ai-test-spicepod
+
+datasets:
+  - from: s3://spiceai-demo-datasets/taxi_trips/2024/
+    name: taxi_trips
+    description: taxi trips in s3
+    params:
+      file_format: parquet
+
+  - from: github:github.com/spiceai/spiceai/issues
+    name: issues
+    description: GitHub Issues for the Spice.ai project (github.com/spiceai/spiceai)
+    metadata:
+      instructions: Always provide reference links. Use the primary key as `issue_id` to generate reference links.
+      reference_url_template: https://github.com/spiceai/spiceai/issues/<issue_id>
+    params:
+      github_token: ${secrets:GITHUB_TOKEN}
+      github_query_mode: "search"
+    acceleration:
+      enabled: true
+      refresh_retry_max_attempts: 3
+      refresh_sql: "SELECT * FROM issues WHERE created_at='2024-11-09T22:47:45'"
+    embeddings:
+      - column: body
+        use: intfloat_e5
+        column_pk:
+          - id
+        chunking:
+          enabled: true
+          target_chunk_size: 512
+          overlap_size: 128
+          trim_whitespace: false 
+    
+  - from: s3://spiceai-public-datasets/tpcds/catalog_page/
+    name: catalog_page
+    description: Represents an e-commerce platform's catalog page metadata
+    params:
+      file_format: parquet
+    acceleration:
+      enabled: true
+      refresh_sql: SELECT * FROM catalog_page limit 20;
+    embeddings:
+      - column: cp_description
+        column_pk:
+          - cp_catalog_page_sk
+        chunking:
+          enabled: true
+          target_chunk_size: 512
+          overlap_size: 128
+          trim_whitespace: false  
+        use: intfloat_e5
+    
+models:
+  - name: hf_local_model
+    from: huggingface:huggingface.co/microsoft/Phi-3-mini-4k-instruct
+    params:
+      model_type: phi3
+      spice_tools: auto
+      system_prompt: |
```

Comment:
Just curious about how the system prompts are generated, they are well written! Did you write it yourself, or there's other tools / reference used.

Reply:
Was taken from Spice with the OpenAI SDK sample:
https://github.com/spiceai/samples/blob/b782524f5850bc4495d8b52d618081dcd1080c19/spice_openai_sdk/spicepod.yaml